components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -101,4 +101,18 @@
  */
 - (void)setContentOffsetY:(CGFloat)contentOffsetY animated:(BOOL)animated;
 
+/**
+ Animates the navigation drawer to the full height of it's content, if the content is larger than
```

Comment:
is this true? it is supposed to animate the drawer to full screen no matter the content height